### PR TITLE
VAULT: Added CLI Commands + Created Test Vault and Operator

### PR DESCRIPTION
**Pull request by coachchucksol**
- Added new commands to the CLI for Vault management
- Created a new document in the /cli folder called `getting_started.md` which walks through the CLI to create and manage a vault

### CODE REVIEW

File: cli/getting_started.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+
+- `<RPC_URL>`: RPC url
+- `<VAULT>`: The vault Pubkey
+- `[NCN_EPOCH]`: Optional NCN Epoch, for closing older, still-open, vault update state trackers
+
+```bash
+jito-restaking-cli --rpc-url <RPC_URL> vault vault close-vault-update-state-tracker <VAULT> <OPERATOR> [NCN_EPOCH]
+```
+
+## Vault Functions
```
Comment (buffalu) [inline cli/getting_started.md:99]: no strong feelings either way, but there is 00_cli.md which contains a bunch of stuff. i think makes more sense for this doc to be focused on creating + updating vault. can fill out as needed

Comment (coachchucksol) [reply cli/getting_started.md:99]: The point of this doc is to add some more context around the different operations and ordering needed for vault creation and maintenance. For example, it explains the vault update loop as well as all of the steps to create and delegate to an operator. I think it's a helpful resource, that being said, I don't care where it lives. It seems like that `00_cli.md` is generated, so I didn't want to put things there.

**Review by buffalu — APPROVED** [review]

File: cli/src/vault.rs [hunk 2]
```diff
@@ -53,6 +53,91 @@ pub enum VaultActions {
...
+        vault: String,
+    },
+    /// Gets the update state tracker for a vault
+    GetVaultUpdateStateTracker {
+        /// Vault account
+        vault: String,
+        /// NCN epoch
+        ncn_epoch: u64,
+    },
+    /// Gets the
```
Comment (ebatsell) [inline cli/src/vault.rs:128]: nit: incomplete message

**Review by ebatsell — APPROVED** [review]
🚀

File: cli/src/vault.rs (hunk 2 above)
Comment (coachchucksol) [reply cli/src/vault.rs]: done

**Pull request merged**